src/test/java/com/google/api/codegen/testsrc/common/library.proto

```diff
@@ -368,8 +369,8 @@ service LibraryService {
   }
 
   // This method is not exposed in the GAPIC config. It should be generated.
-  rpc PrivateGetBook(GetBookRequest) returns (Book)  {
-    option (google.api.http) = { get: "/v1/{name=bookShelves/*/books/*}" };
+  rpc PrivateListShelves(ListShelvesRequest) returns (Book)  {
```

Comment:
The old `PrivateGetBook` method was used only to test that an RPC defined in the protofile but not in the GAPIC config would still be generated.

This new `PrivateListShelves` method does the same thing, in addition to testing that a  request object method that lacks a `page_size` field will not be generated as a paginated method (ListShelvesRequest does not have a `page_size` member).